vignettes/cookbook.Rmd

```diff
@@ -36,68 +37,72 @@ Line graphs are useful for showing change over time, and are well-suited for com
 
 #### Example code
 
-```{r line, message=FALSE}
-# get example dataset, and create a relative GRP field
+```{r line, fig.asp=4.5/(670/72), message=FALSE}
```

Comment:
@matthewstern FYI, this had to be set in order to use `mode="plot"` and `height=4.5` in `finalize_plot()` without grey bars on the top and bottom of the image.